### PR TITLE
Adds OpenAPI docs

### PR DESCRIPTION
**Pull request by SweeXordious**

### CODE REVIEW

**Comment by SweeXordious** [comment]
@conor10 Can you please review this ?

File: docs/quickstart.md [hunk 1]
```diff
@@ -173,18 +176,18 @@ epirus docker run -l <network>
...
+- `WEB3J_WALLET_PATH`
   Path to Ethereum wallet
-- `EPIRUS_WALLET_PASSWORD`
+- `WEB3J_WALLET_PASSWORD`
   Ethereum wallet password
 
 For an OpenAPI service, you will need to define the following:
 
-- `WEB3J_OPENAPI_PRIVATE_KEY`
+- `WEB3J_PRIVATE_KEY`
```
Comment (conor10) [inline docs/quickstart.md:186]: These should be the same for a Web3j app or OpenAPI app - they're here twice

File: docs/quickstart.md [hunk 2]
```diff
@@ -219,7 +222,7 @@ For OpenAPI services, the following properties can be used:
   Ethereum node URL
 - `WEB3J_PRIVATE_KEY`
   Hex-encoded private key string (0x...) 
-- `WEB3J_WALLET_FILE`
```
Comment (conor10) [inline docs/quickstart.md:222]: Again we're repeating ourselves

File: docs/sdk/cli.md [hunk 3]
```diff
@@ -310,17 +267,24 @@ Solidity `bin` and `abi` files are located at:
...
 
 ## Generated OpenAPI project structure
 
-Currently OpenAPI projects are not intended to be modified once generated. I.e. they are only meant to be run by the user to deploy and interact directly with a smart contract.
+Similar to the Java/Kotlin projects. The Solidity files are located in the following `./src/main/solidity`.
 
-We expect to be releasing an Web3j-OpenAPI Gradle plugin shortly that will enable users to easily modify the generated services.
+Additionally, the generated OpenAPI code resides in `./build/generated/sources/web3j/main`, and is structured as follows :
+ 
+- Java wrappers :  `java/<package name>/wrappers`
```
Comment (conor10) [inline docs/sdk/cli.md:282]: Java smart contract wrappers

File: docs/sdk/cli.md [hunk 4]
```diff
@@ -349,7 +325,7 @@ To delete all project build artifacts, creating a clean environment, run:
 When creating a new project or importing solidity contracts, by using:
 
 ``` shell
-epirus generate-tests
+epirus generate tests
```
Comment (conor10) [inline docs/sdk/cli.md:328]: Is this right?

File: docs/sdk/cli.md [hunk 5]
```diff
@@ -210,30 +167,30 @@ A number of properties can be configured for your Epirus applications to customi
 
 The following configuration properties can be used for Java or Kotlin projects:
```
Comment (conor10) [inline docs/sdk/cli.md:168]: We should say Java or OpenAPI projects here

File: docs/sdk/cli.md [hunk 6]
```diff
@@ -210,30 +167,30 @@ A number of properties can be configured for your Epirus applications to customi
...
-- `WEB3J_OPENAPI_CONTEXT_PATH`
-  Project context path, defaults to project name 
+- `WEB3J_OPENAPI_CONTRACT_ADDRESSES`
+  Pre-deployed contract addresses as a map (Contract1=0x...,Contract2=0x...)
 - `WEB3J_OPENAPI_HOST`
   Hostname for service, defaults to localhost
 - `WEB3J_OPENAPI_PORT`
   Port to bind to, defaults to 9090
-- `WEB3J_OPENAPI_ENDPOINT`
+- `WEB3J_ENDPOINT`
```
Comment (conor10) [inline docs/sdk/cli.md:187]: Then we can get rid of this

**Review by conor10 — COMMENTED** [review]
We need to minimise how much we're repeating the WEB3J environment variables you can use. I would suggest we only have them once in the quickstart and once in the main docs.

File: docs/sdk/cli.md (hunk 4 above)
Comment (SweeXordious) [reply docs/sdk/cli.md:328]: yes, the generation commands were refactored

**Comment by SweeXordious** [comment]
@conor10 what do you think now ?

**Review by conor10 — APPROVED** [review]
I'm going to get on with the merge - I have someone who needs this to work with now.

**Pull request merged**